notification/bandwidth.py

```
@@ -0,0 +1,158 @@
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: bandwidth
+version_added: 2.2
+short_description: Send a sms / mms using the bandwidth api
+description:
+     - The M(bandwidth) module allows notifications to be sent using sms / mms via the bandwidth api.
+options:
+  src:
+    description:
+      - One of your bandwidth telephone numbers the message should come from (must be in E.164 format, like +19195551212).
+    required: true
+    default: null
+  dest:
+    description:
+      - The phone number or numbers the message should be sent to (must be in E.164 format, like +19195551212).
+    required: true
+    default: null
+  msg:
+    description:
+      - The contents of the text message (must be 2048 characters or less).
+    required: true
+    default: null
+  media:
+    description:
+      - For MMS messages, a media url to the location of the media to be sent with the message.
+    required: false
+    default: null
+  user_id:
+    description:
+      - User Id from Api account page
+    required: true
+    default: null
+  api_token: 
+    description:
+      - Api Token from Api account page
+    required: true
+    default: null
+  api_secret: 
+    description:
+      - Api Secret from Api account page
+    required: true
+    default: null
+
+author: "Jonathan Mainguy (@Jmainguy)"
+notes:
+    - Will return changed even if the media url is wrong
+    - Will return changed if the destination number is invalid
```

Should this be a `fail` instead? Seems like silently letting numbers be invalid could lead to issues.